ompi/peruse/peruse.h

```diff
@@ -36,7 +36,7 @@ typedef int (peruse_comm_callback_f)(peruse_event_h event_h,
 
 /* PERUSE constants */
 enum {
-    PERUSE_SUCCESS = 0,       /* Success *//* XXX Devation from 1.11 */
+    PERUSE_SUCCESS = 0,       /* Success *//* XXX Deviiation from 1.11 */
```

Comment:
Is this right?  Shouldn't it be "Deviation" (one "i", not two)?

Reply:
Apologies, not sure how that snuck in. Fixed in [aa5a3d3](https://github.com/open-mpi/ompi/pull/10607/commits/aa5a3d36a665dae4a213b7094448e694d85ff4e0)